oe1176/DynSports/PASOEContent/WEB-INF/openedge/Spark/Core/Handler/StatusHandler.cls

```diff
@@ -110,7 +110,7 @@ class Spark.Core.Handler.StatusHandler inherits OpenEdge.Web.WebHandler use-widg
                 oWriter:write('</table>').
                 oWriter:write('</section>').
 
-                oWriter:write('<footer>Copyright &copy; 2020 Progress Software Corporation. All rights reserved.</footer>').
+                oWriter:write('<footer>Copyright &copy; ' + string(year(today)) + ' Progress Software Corporation. All rights reserved.</footer>').
```

Comment:
Same comment as in the main Spark repo - should this be written as the run date, not the 'dev on' date?

Reply:
It's a piece of demo code, and I didn't want the HTML output using an outdated date. Either that or I remove it entirely.